### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RudarDaman**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by Toxicable** [comment]
The current version isn't 2

**Comment by trotyl** [comment]
Did you mixed up **Angular** and **AnguarJS**?

[Branding Guidelines for Angular and AngularJS](http://angularjs.blogspot.hk/2017/01/branding-guidelines-for-angular-and.html)
[Angular - PRESS KIT - Brand Names](https://angular.io/presskit#brand-names)

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 # Angular
 
-Angular is a development platform for building mobile and desktop web applications using Typescript/JavaScript and other languages.
+Angular is a development platform for building mobile and desktop web applications using Typescript/JavaScript and other languages.The current version is 2.
```
Comment (gkalpak) [inline README.md]: No it's not :smiley: It is 4 right now and will be 5 in less than a week. The point is, [it's just Angular][1] :smiley:

[1]: http://angularjs.blogspot.gr/2016/12/ok-let-me-explain-its-going-to-be.html

Comment (akhilesh-k) [reply README.md]: Maybe done just for Hacktober fest :laughing:

**Pull request closed without merging**